### PR TITLE
EVG-2166: add amboy stats collection to the runner

### PR DESCRIPTION
**Pull request by tychoish**

### CODE REVIEW

File: cli/runner.go [hunk 1]
```diff
@@ -94,8 +96,39 @@ func (c *ServiceRunnerCommand) Execute(_ []string) error {
...
+					"stats":   remoteq.Stats(),
+					"report":  amboy.Report(ctx, remoteq, 512),
+				})
+			}
+		}
+	}
+}
+
+func taskStatsCollector(ctx context.Context) {
+	defer recovery.LogStackTraceAndContinue("amboy stats collector")
```
Comment (john-m-liu) [inline cli/runner.go]: the logging comments are reversed with the stats collector

File: cli/runner.go [hunk 2]
```diff
@@ -94,8 +96,39 @@ func (c *ServiceRunnerCommand) Execute(_ []string) error {
...
+	const interval = time.Minute
+	timer := time.NewTimer(0)
+	defer timer.Stop()
+	for {
+		select {
+		case <-ctx.Done():
+			grip.Info("task stats logging operation canceled")
+			return
+		case <-timer.C:
+			if localq := env.LocalQueue(); localq.Started() {
```
Comment (bsamek) [inline cli/runner.go]: Declare this variable (and on 119) before the loop begins?

File: cli/runner.go [hunk 3]
```diff
@@ -94,8 +96,39 @@ func (c *ServiceRunnerCommand) Execute(_ []string) error {
...
+				grip.Info(message.Fields{
+					"message": "amboy local queue stats",
+					"stats":   localq.Stats(),
+					"report":  amboy.Report(ctx, localq, 512),
```
Comment (bsamek) [inline cli/runner.go]: Make 512 a constant (and on line 123).

File: cli/runner.go (hunk 2 above)
Comment (bsamek) [reply cli/runner.go]: Actually, thought about this more, this is fine.

**Comment by tychoish** [comment]
fixed wrt brian's comments

**Pull request merged**